topdown/example_test.go

```diff
@@ -25,7 +27,10 @@ func ExampleEval() {
 
 	var data map[string]interface{}
 
-	if err := json.Unmarshal([]byte(`{"a": [1,2,3,4]}`), &data); err != nil {
+	decoder := json.NewDecoder(bytes.NewBufferString(`{"a": [1,2,3,4]}`))
+	decoder.UseNumber()
```

Comment:
Not opposed to pointing readers at util.UnmarshalJSON but I was thinking it's better to show `UseNumber` explicitly in the examples.